### ISSUE TITLE
Enable Cloud Provider Config file as a parameter during driver installation, needed for ARO support

### ISSUE THREAD

**Issue by ezYakaEagle442** [kubernetes-sigs/azuredisk-csi-driver#398]
**What happened**:

The [CSI driver doc](https://github.com/kubernetes-sigs/azuredisk-csi-driver#prerequisite) states : 
The driver initialization depends on a Cloud provider config file, usually it's /etc/kubernetes/azure.json on all kubernetes nodes deployed by AKS or aks-engine, here is azure.json example. This driver also supports read cloud config from kuberenetes secret.
 
Indeed this is what I see in the [HELM Chart](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/charts/v0.7.0/azuredisk-csi-driver/templates/csi-azuredisk-node.yaml#L93) 

**What you expected to happen**:

The driver should be installed correctly in Azure RedHat OpenShift 4.3

**How to reproduce it**:

I am testing the CSI driver for Azure disk on ARO 4.3, snippet in progress available at [https://github.com/ezYakaEagle442/aro-pub-storage/blob/master/setup-store-CSI-driver-azure-disk.md](https://github.com/ezYakaEagle442/aro-pub-storage/blob/master/setup-store-CSI-driver-azure-disk.md) 
```sh
oc get secrets -n kube-system
oc describe secret azure-cloud-provider -n kube-system
azure_cnf_secret=$(oc get secret azure-cloud-provider -n kube-system -o jsonpath="{.data.cloud-config}" | base64 --decode)
echo "Azure Cloud Provider config secret " $azure_cnf_secret
```
Then the logs show the azuredisk container fails to read the file /etc/kubernetes/azure.json :
```sh
for pod in $(oc get pods -l app=csi-azuredisk-node -n kube-system -o custom-columns=:metadata.name)
do
    oc logs $pod -c azuredisk -n kube-system # | grep -i "Error"
done
```
I0518 19:17:53.082115       1 azure_publicipclient.go:66] Azure PublicIPAddressesClient (write ops) using rate limit config: QPS=1, bucket=5
I0518 19:17:53.082150       1 azure.go:50] could not read cloud config from secret
I0518 19:17:53.082160       1 azure.go:53] **AZURE_CREDENTIAL_FILE env var set as /etc/kubernetes/azure.json**
E0518 19:17:53.082175       1 azure.go:61] **Failed to load config from file: /etc/kubernetes/azure.json**
F0518 19:17:53.082180       1 azuredisk.go:103] **failed to get Azure Cloud Provider, error: Failed to load config from file: /etc/kubernetes/azure.json, cloud not get azure cloud provider**

**Anything else we need to know?**:

The above [DS runs with SA ](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/charts/v0.7.0/azuredisk-csi-driver/templates/csi-azuredisk-node.yaml#L18) csi-azuredisk-node-sa :

I added this SCC but I still got the same error after a clean uninstall / reinstall :
 ```sh
oc adm policy add-scc-to-user privileged system:serviceaccount:kube-system:csi-azuredisk-node-sa
oc describe scc privileged
 ```
 
In openshift the creds file is located in “/etc/kubernetes/**cloud.conf**”, so you would need to replace the path in the deployment for the driver from “/etc/kubernetes/azure.json” to “/etc/kubernetes/cloud.conf”.

[CSI driver doc states](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/docs/read-from-secret.md) : This driver also supports reading the cloud config from Kubernetes secrets. The secret is a serialized version of **azure.json file with key cloud-config. The secret should be put in kube-system namespace and its name should be azure-cloud-provider.**

see similar issue with Azure File Driver : [https://github.com/kubernetes-sigs/azurefile-csi-driver/issues/282](https://github.com/kubernetes-sigs/azurefile-csi-driver/issues/282)

**Environment**:
- CSI Driver version: v0.7.0 
- Kubernetes version (use `kubectl version`): ARO 4.3 ==> openshift-clients-4.3.14-202004180552 , Server Version: 4.3.18, Kubernetes Version: v1.16.2
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by andyzhangx** [comment]
thanks for raising this issue, I think use helm should be the right way to set a different cloud provider config file.

**Comment by andyzhangx** [comment]
already fixed by #400 
And you could use following workaround to install CSI drivers on prior releases, e.g. v0.7.0
```
git clone https://github.com/kubernetes-sigs/azuredisk-csi-driver.git
cd azuredisk-csi-driver
sed -i 's/azure.json/cloud.conf/g' deploy/v0.7.0/*.yaml
./deploy/install-driver.sh v0.7.0 local
```
